Sprinkle of Yeezus/Settings.swift

```diff
@@ -12,67 +12,185 @@ import UserNotifications
                 print(error)
+                }
             }
         }
+        updateLabelTime()
+    }
+    
+    private func anniversaryNotifications() {
+        guard notificationSwitch.isOn else { //Runs only user turns on notification switch
+            return
+        }
+        
+        let today = Date()
+        let currentCalendar = Calendar.current
+        var anniversaryDate = DateComponents()
+        let notification = UNMutableNotificationContent()
+        
+        struct album{
+            let name: String
+            let releaseDate: DateComponents
+        }
+        
+        let albumList = [ 
+            album(name: "The College Dropout", releaseDate: DateComponents(year: 2004, month: 2, day: 10)),
+            album(name: "Late Registration", releaseDate: DateComponents(year: 2005, month: 8, day: 30)),
+            album(name: "Graduation", releaseDate: DateComponents(year: 2007, month: 9, day: 11)),
+            album(name: "808's and Heartbreak", releaseDate: DateComponents(year: 2008, month: 11, day: 24)),
+            album(name: "My Beautiful Dark Twisted Fantasy", releaseDate: DateComponents(year: 2010, month: 11, day: 22)),
+            album(name: "Yeezus", releaseDate: DateComponents(year: 2013, month: 6, day: 18)),
+            album(name: "The Life of Pablo", releaseDate: DateComponents(year: 2016, month: 2, day: 14)),
+            album(name: "Ye", releaseDate: DateComponents(year: 2018, month: 6, day: 1)),
+            album(name: "Watch the Throne", releaseDate: DateComponents(year: 2011, month: 8, day: 8)),
+            album(name: "Cruel Summer", releaseDate: DateComponents(year: 2012, month: 9, day: 14)),
+            album(name: "Kids See Ghosts", releaseDate: DateComponents(year: 2018, month: 6, day: 8))
+            ]
+        
+        for item in albumList{
+            anniversaryDate.year = currentCalendar.component(.year, from: today)
+            anniversaryDate.month = item.releaseDate.month
+            anniversaryDate.day = item.releaseDate.day
+            
+            let albumAge = anniversaryDate.year! - item.releaseDate.year! //time from albums release to current year
+            notification.body = "🎉 \(item.name) was released \(albumAge) years ago today! 🎂"
+            let trigger = UNCalendarNotificationTrigger(dateMatching: anniversaryDate, repeats: false)
+            let request = UNNotificationRequest(identifier: "Album anniversary for \(item.name)", content: notification, trigger: trigger)
+            
+            print("The anniversary notification for \(item.name) has been schedualed on \(anniversaryDate)")
+            
+            notificationCenter.add(request) { error in
+                if let error = error {
+                    print(error)
+                }
+            }
+        }
+        
+    }
+    
+    func updateSwitchStatus(){ //this function checks to see if the switch was flipped in the past and acts accordingly
```

Comment:
Add this comment directly above the function and use 3 slashes `///` and that indicates documentation. So that when you ctrl + click on a function it will show the comment